ivy/functional/frontends/jax/numpy/logic.py

```diff
@@ -283,3 +283,28 @@ def packbits(x, /, *, axis=None, bitorder="big"):
     bits = ivy.expand_dims(bits, axis=tuple(range(x.ndim - 1)))
     packed = (x << bits).sum(axis=-1).astype("uint8")
     return ivy.swapaxes(packed, axis, -1)
+
+
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes({"0.4.14 and below": ("bool",)}, "jax")
```

Comment:
Hi @Sarvesh-Kesharwani 
you're again restricting the dtypes, I think we talked about this already.

Reply:
Hi @abdul, about the fromfunction https://github.com/unifyai/ivy/pull/21432,

although you are correct that i should not restrict this func for bool dtype but upon testing i found that my implementation which internally uses ivy's gather_nd function which doesn't support bfloat16 dtype, tell me what you think? 
![image](https://github.com/unifyai/ivy/assets/22115102/593594f7-905c-440a-9d36-97de2a62f142)